airbyte-config/models/src/main/resources/types/StandardDestinationDefinition.yaml

```diff
@@ -34,6 +34,14 @@ properties:
       if not set or false, the configuration is active. if true, then this
       configuration is permanently off.
     type: boolean
+  public:
+    description: true if this connector definition is available to all workspaces
+    type: boolean
+    default: false
+  custom:
```

Comment:
What is the definition of custom vs public? I would be careful before adding that kind of specific flags on the data model. They risk proliferating.
Are they mutually exclusive? then it should be a enum or just one boolean
Can it be true on both? false on both? what is the expected behavior in these cases?

Reply:
Great points.  Let me copy this over to #11305 (which has these same migration and model changes broken out into its own PR) and continue the discussion there.
Worth discussing since there was a thread about this in the design doc as well